src/main/java/org/maxgamer/quickshop/integration/FabledSkyblock/FabledIntegration.java

```diff
@@ -0,0 +1,100 @@
+package org.maxgamer.quickshop.integration.fabledskyblock;
+
+import com.songoda.skyblock.SkyBlock;
+import com.songoda.skyblock.island.Island;
+import com.songoda.skyblock.island.IslandRole;
+import org.bukkit.Location;
+import org.bukkit.entity.Player;
+import org.bukkit.event.Listener;
+import org.jetbrains.annotations.NotNull;
+import org.maxgamer.quickshop.QuickShop;
+import org.maxgamer.quickshop.integration.IntegrateStage;
+import org.maxgamer.quickshop.integration.IntegrationStage;
+import org.maxgamer.quickshop.integration.QSIntegratedPlugin;
+
+@IntegrationStage(loadStage = IntegrateStage.onEnableAfter)
+public class FabledIntegration extends QSIntegratedPlugin implements Listener {
+
+    private final boolean ignoreDisabledWorlds;
+    private final boolean whitelist;
+
+    public FabledIntegration(QuickShop plugin) {
+        super(plugin);
+        ignoreDisabledWorlds = plugin.getConfig().getBoolean("integration.fabledskyblock.ignore-disabled-worlds");
```

Comment:
Please add config update code in org.maxgamer.quickshop.QuickShop#updateConfig, also update `config-version` and config node in `config.yml`.